src/lib/y2network/sysconfig/config_writer.rb

```diff
@@ -103,20 +101,22 @@ def write_interface_changes(config, old_config)
       #
       # @param routing [Y2Network::Routing] routing configuration
       def write_ip_forwarding(routing)
-        sysctl = CFA::Sysctl.new
-        sysctl.load
-        sysctl.forward_ipv4 = routing.forward_ipv4
-        sysctl.forward_ipv6 = routing.forward_ipv6
-        sysctl.save
-
-        update_ip_forwarding(sysctl.raw_forward_ipv4, :ipv4)
-        update_ip_forwarding(sysctl.raw_forward_ipv6, :ipv6)
+        sysctl_config = CFA::SysctlConfig.new
+        sysctl_config.load
+        sysctl_config.forward_ipv4 = routing.forward_ipv4
+        sysctl_config.forward_ipv6 = routing.forward_ipv6
+        sysctl_config.save unless sysctl_config.conflict?
+
+        update_ip_forwarding((sysctl_config.forward_ipv4 ? "1" : "0"),
```

Comment:
Why were modified the calls to `raw_forward_` methods ? are them not valid anymore?

Reply:
Imo has wished to remove them in the new class. :-)

Reply:
Ok, then next time we touch the class we should also get rid of of this documentation that is out of date or never implemented for this new class :stuck_out_tongue_winking_eye: 

https://github.com/yast/yast-yast2/pull/1021/files#diff-4216cbac8e98ed10343a00513e10e917R47